marimbabot_audio/launch/marimbabot.launch

```diff
@@ -1,14 +1,31 @@
 <?xml version="1.0"?>
 <launch>
+
+  <!-- list the name of devices: pactl list | grep -A2 'Source #' | grep 'Name: ' | cut -d" " -f2 | grep input -->
+<<<<<<< HEAD
     <node name="audio_from_lilypond" pkg="marimbabot_audio" type="audio_from_lilypond.py"/>
-    <node name="audio_audio_capture" pkg="audio_capture" type="audio_capture" output="screen">
-        <param name="device" value="" />
-        <param name="format" value="wave" />
-        <param name="channels" value="1" />
-        <param name="depth" value="16" />
-        <param name="sample_rate" value="44100" />
-    </node>
+=======
+>>>>>>> edd50390e426be44db374a26a50dbddac9826170
```

Comment:
It seems like you committed a merge conflict

Reply:
working on it